scripts/configs/system.go

```diff
@@ -0,0 +1,99 @@
+package configs
+
+import (
+	"fmt"
+	"runtime"
+)
+
+// System environment struct
+type SystemEnvironmentStruct struct {
+	GoInstalled                                 bool
```

Comment:
Instead of hardcoding the values in the code, we should have a JSON file where users can tweak the parameters (the default file can be in the repo). 

Example of such implementation is the usage of https://github.com/eth-easl/loader/blob/main/cmd/config.json by the loader repo